Updates for schedule api changes (#1595)
temporalio/documentation: docs/tctl/schedule/list.md

```

Note that if you're using Standard Visibility, listing Schedules will currently only include Schedule Ids and no other information.

> The Schedule Specs in the output are canonicalized and might not be in the same form as they were input.

Reviewer: ```suggestion
Because the Schedule Spec is converted to canonical representations, the output might not be in the same form as it was input.
```
